README.md

```diff
@@ -314,7 +314,7 @@ To learn more about making a contribution to NNI, please refer to our [How-to co
 
 We appreciate all contributions and thank all the contributors!
 
-<a href="https://github.com/microsoft/nni/graphs/contributors"><img src="https://contrib.rocks/image?repo=microsoft/nni" /></a>
+<a href="https://github.com/microsoft/nni/graphs/contributors"><img src="https://contrib.rocks/image?repo=microsoft/nni&max=240" /></a>
```

Comment:
Did we discussed about making this image look smaller?

Reply:
Not my task I guess 🙂